gui/resources.xml

```diff
@@ -763,27 +784,35 @@ Vocabulary
       <!-- {{{ inktool points related actions -->
       <child>
         <object class="GtkAction" id="InsertCurrentNode">
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a node back of the currently selected node.</property>
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Insert Inking Node</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Insert a inking node on the back of the currently selected node.</property>
           <signal name="activate" handler="insert_current_node_cb"/>
         </object>
+        <accelerator key="Insert"/>
       </child>
       <child>
         <object class="GtkAction" id="DeleteCurrentNode">
-          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Delete currently selected node.</property>
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Delete Inking Node</property>
+          <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Delete currently selected node on the inking tool.</property>
           <signal name="activate" handler="delete_current_node_cb"/>
         </object>
+        <accelerator key="Insert" modifiers="GDK_SHIFT_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="SimplifyNodes">
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Simplify Inking Nodes</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Simplify inking tool nodes according to angle and distance between nodes.</property>
           <signal name="activate" handler="simplify_nodes_cb"/>
         </object>
+        <accelerator key="Insert" modifiers="GDK_SHIFT_MASK | GDK_CONTROL_MASK"/>
       </child>
       <child>
         <object class="GtkAction" id="CullNodes">
+          <property name="label" translatable="yes" context="Accel Editor (labels)">Cull Inking Node</property>
           <property name="tooltip" translatable="yes" context="Accel Editor (descriptions)">Cull inking tool nodes alternately.</property>
           <signal name="activate" handler="cull_nodes_cb"/>
         </object>
+        <accelerator key="Insert" modifiers="GDK_CONTROL_MASK | GDK_MOD1_MASK"/>
```

Comment:
See the comment above. Currently they're only accessible via the panel, which is why they don't have labels.

It's fine to add labels though - they just won't be seen!

"Simplify Ink Points"?